base/array.jl

```diff
@@ -64,10 +64,17 @@ Two-dimensional dense array with elements of type `T`, often used to represent
 a mathematical matrix. Alias for [`Array{T,2}`](@ref).
 """
 const Matrix{T} = Array{T,2}
+
 """
     VecOrMat{T}
 
 Union type of [`Vector{T}`](@ref) and [`Matrix{T}`](@ref).
+
+# Examples
+```jldoctest
+julia> VecOrMat{Float64}
+Union{Array{Float64,1}, Array{Float64,2}}
```

Comment:
```suggestion
julia> Vector{Float64} <: VecOrMat{Float64}
true

julia> Matrix{Float64} <: VecOrMat{Float64}
true

julia> Array{Float64,3} <: VecOrMat{Float64}
false
```
Julia v1.6 displays aliases, so the original example is a kind of tautology. The tautology itself isn't that bad, but as you've given an example, the doctest is fragile.

Reply:
Great, adding in the example you suggested.